src/apppttwo.js

```diff
@@ -0,0 +1,60 @@
+//*************************************** PART 2 **********************************************************************
+
+import axios from "axios";
+
+const countrySearchResult = document.getElementById("countryResult");
+const errorMessage = document.getElementById("error");
+
+const searchCity = document.getElementById("searchButton");
+searchCity.addEventListener('submit', sendData);
+
+function sendData(e) {
+    e.preventDefault();
+    const valInput = document.getElementById('city-input');
+    fetchCountryData(valInput.value);
+    valInput.value = ""
+}
+
+
+async function fetchCountryData(name) {
+    countrySearchResult.innerHTML = ``;
+    try {
+        const resultCountry = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const country = resultCountry.data[0];
+        errorMessage.innerHTML = "";
+        listingItTwo(country);
+    } catch (e) {
+        console.error(e);
+        errorMessage.innerHTML = `
+        <h2>Oepsie woepsie!</h2>
+        <p>De site is stukkie-wukkie!</p>
+        <p>Ziet ernaar uit dat er iets mis is gegaan.</p>
+        <p>Is de invoer correct?</p>
+        `
+    }
+}
+
+function listingItTwo(country) {
```

Comment:
Misschien dat je hier object destructering toe kan passen:
```suggestion
function listingItTwo(({name, flag, subregion, population, currencies, capital})) {
```

Dan hoef je niet overal `country.` voor te zetten